pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -65,6 +67,9 @@
 , systemdSupport ? stdenv.isLinux
 }:
 
+let
+  cmakeBool = x: if x then "ON" else "OFF";
```

Comment:
Why this change?

Reply:
Oh, this was moved just for nix formatting. I will change it back.